sklearn/manifold/_t_sne.py

```diff
@@ -700,6 +704,9 @@ class TSNE(BaseEstimator):
 
         .. versionadded:: 1.0
 
+    learning_rate_ : float
+        Effective learning rate.
```

Comment:
+1 for exposing this in the public API, this is useful. But please add a `versionadded`.